.github/workflows/ci.yml

```diff
@@ -15,9 +15,11 @@ concurrency:
   cancel-in-progress: true
 
 env:
+  GIT_BRANCH: ${{ github.head_ref || github.ref_name }}
   CARGO_TERM_COLOR: always
   RUST_VERSION: 1.75.0
-  NIGHTLY_RUST_VERSION: nightly-2023-10-29
+  RUST_VERSION_FMT: nightly-2023-10-29
+  RUST_VERSION_COV: nightly-2024-06-05
```

Comment:
Why do we use a different version of rust nightly?

Reply:
I didn't want to touch formatting in this PR. The branch coverage needs.a more recent nightly.

Reply:
Okay. That makes sense. Do we have any issues for unifying them?

Reply:
In a separate PR, maybe. I don't want to reformat the whole codebase here.